ui/web_remote/favicon/manifest.json

```diff
@@ -0,0 +1,41 @@
+{
+ "name": "App",
+ "icons": [
+  {
+   "src": "\/android-icon-36x36.png",
```

Comment:
I didn't actually generate this. Used an online tool to generate all the required files from an initial PNG file.